bootstrap/environment/variables.go

```diff
@@ -233,6 +233,11 @@ func (e *Variables) buildPaths(keyMap map[string]any) []string {
 	var paths []string
 
 	for key, item := range keyMap {
+		if item == nil {
+			fmt.Printf("Invalid value (<nil>) for key \"%s\". Skipping.", key)
+			continue
+		}
+		
 		if reflect.TypeOf(item).Kind() != reflect.Map {
```

Comment:
This is the more appropriate fix. 

```suggestion
		if item == nil || reflect.TypeOf(item).Kind() != reflect.Map {
```